common/artifact-manager/src/main/java/com/vmware/pscoe/iac/artifact/VroPackageStore.java

```diff
@@ -375,4 +393,31 @@ private String createBackupFilePath(final Package pkg, final String currentDateT
 
 		return newFullPath;
 	}
+
+	/**
+	 * Compares packages by their version and return the highest version package.
+	 * @param packages the collection of packages differring by version
+	 * @return the package with highest version
+	 */
+	private Package getHighestVersion(List<Package> packages) {
```

Comment:
Why do we want to have only the latest versions backed up? We should be receiving backup of all versions, as Build Tools for VMware Aria - might have the cleanup flag enabled and delete some of the versions that we did not backed up.

Reply:
Following the discussion in the [jira](https://jira.pscoe.vmware.com/browse/IAC-778) issue, this is what was finally agreed.